client/deposits/utils/index.js

```diff
@@ -0,0 +1,137 @@
+/**
```

Comment:
Some tests are missing for this file as they were not present in the first place. Maybe we can include those tests in a separate PR later. I'll leave that up to you to decide.

Reply:
Here they are: https://github.com/Automattic/woocommerce-payments/pull/1892/commits/fa76c4aa9c99a73a36c9e7609867b3a6217fa9e7 :)

Reply:
Thank you! so much for this ❤️ .